js/src/merkle.ts

```diff
@@ -144,10 +145,10 @@ export class MerkleTree {
     }
 }
 
-export function loadTree(balances) {
+export function loadTree(balances, decimals = 18) {
     const elements: any = [];
     Object.keys(balances).forEach((address) => {
-        const balance = toWei(balances[address]);
+        const balance = scale(balances[address], decimals);
```

Comment:
Apparently `scale` doesn't suppress the decimal places in excess. So if `balance[address]` has 12 decimals and `decimals == 10`, we get eg this error

```
Error: Error: [number-to-bn] while converting number "3555529890357.96" to BN.js instance, error: invalid number value. Value must be an integer, hex string, BN or BigNumber instance. Note, decimals are not supported. Given value: "3555529890357.96"
```

Maybe we can replace this line with something like this?
```
const balance = scale(balances[address], decimals).integerValue(BigNumber.ROUND_DOWN);
```

Or we refactor `scale`.

Reply:
I think that's actually a useful error to leave as a sanity check.  Wondering if we can rerun the distribution so that `balance[address]` has 10 decimals as expected?